From a55a216498b474c7b5c20d3262e7fa56c43ae05b Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 5 May 2015 10:47:55 +0800
Subject: [PATCH 1040/1594] MLK-10815-2 ARM: dts: add ASRC support for
 imx6ul-evk board

commit 963f9f94a45f387bf800e53365cdeb984bb01998 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add ASRC support for imx6ul-evk board.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk.dts |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
index 517cff0..1ccb451 100755
--- a/arch/arm/boot/dts/imx6ul-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-evk.dts
@@ -42,6 +42,7 @@
 		model = "wm8960-audio";
 		cpu-dai = <&sai2>;
 		audio-codec = <&codec>;
+		asrc-controller = <&asrc>;
 		codec-master;
 		gpr = <&gpr>;
 		/*
-- 
1.7.5.4

